<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="screenLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">);</string>
      </property>
      <property name="title">
       <string>延时选项</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QSpinBox" name="spinBox">
             <property name="styleSheet">
              <string notr="true">background: transparent;
</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">background: transparent;
font: 10pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>延时时间（s)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="styleSheet">
            <string notr="true">background: transparent;
font: 10pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>是否延时</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="newScreenShotButton">
           <property name="styleSheet">
            <string notr="true">background: transparent;
font: 10pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string>新建截图</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="savePictureButton">
           <property name="styleSheet">
            <string notr="true">background: transparent;
color: rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>保存截图</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="quitButton">
           <property name="styleSheet">
            <string notr="true">background: transparent;
color: rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
